import-export-cli/utils/constants.go

```diff
@@ -68,6 +68,7 @@ const DefaultExportDirName = "exported"
 const ExportedApisDirName = "apis"
 const ExportedPoliciesDirName = "policies"
 const ExportedThrottlePoliciesDirName = "rate-limiting"
+const ExportedAPIPoliciesDirName = "operation"
```

Comment:
Let's change this to "apis" as discussed.

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)